tests/test_cancellation/result.txt

```diff
@@ -1,231 +1,231 @@
 Info [interflop_cancellation]: interflop_cancellation: loaded backend with tolerance = 0
```

Comment:
is this file suppose to be comited on the git?

Reply:
Yes, it's the reference to compare against.

Reply:
So why is there some changes in all the ref values despite the benchmark code itself has not change?

Reply:
I believe because the test depends on the random seed. Matei had the same issue when he was working on the new RNG interface.

Reply:
Exactly

> I believe because the test depends on the random seed. Matei had the same issue when he was working on the new RNG interface.

Exactly.